layers/auto-layer.el

```diff
@@ -74,6 +74,7 @@
 (configuration-layer/lazy-install 'sml :extensions '("\\(\\.s\\(ml\\|ig\\)\\'\\|\\.\\(sml\\|sig\\)\\'\\)" sml-mode))
 (configuration-layer/lazy-install 'sql :extensions '("\\(\\.sql\\'\\)" sql-mode))
 (configuration-layer/lazy-install 'swift :extensions '("\\(\\.swift\\'\\)" swift-mode))
+(configuration-layer/lazy-install 'systemd :extensions'("\\(\\.\\(nspawn\\|automount\\|busname\\|mount\\|service\\|slice\\|socket\\|swap\\|target\\|timer\\|link\\|netdev\\|network\\)\\)" systemd-mode))
```

Comment:
This regexp matches strings inside a filename, it should match it only at the end of a filename.

Reply:
You're right, it was missing the `\\'`. Fixed.